src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/ApplicationAppSI.cs

```diff
@@ -14,20 +19,29 @@ namespace Altinn.App.Services.Implementation
     /// </summary>
     public class ApplicationAppSI : IApplication
     {
+        private readonly PlatformSettings _platformSettings;
         private readonly ILogger _logger;
         private readonly HttpClient _client;
 
+        private const string SubscriptionKeyHeaderName = "Ocp-Apim-Subscription-Key";
```

Comment:
I think this can be removed if you are using the value from General instead

Reply:
done